packages/vvisp/test/scripts/deploy-contract.test.js

```diff
@@ -0,0 +1,43 @@
+const deployContract = require('../../scripts/deploy-contract');
+const path = require('path');
+
+describe('# deploy contract process test', function() {
+  const CONTRACT_PATH = path.join('./contracts', 'libs', 'Ownable.sol');
+  const CONTRACT_INHERITED_PATH = path.join(
+    './contracts',
+    'test',
+    'Attachment.sol'
+  );
+  const CONTRACT_FLATTENED_PATH = path.join(
+    './contracts',
+    'test',
+    'Flattend.sol'
+  );
+  const DUMMMY_FILE = path.join('./test', 'dummy', 'test.env');
+
+  it('should deploy one contract', async () => {
+    await deployContract(CONTRACT_PATH, [], { silent: true }).should.be
+      .fulfilled;
+  });
+
+  it('should deploy two contract', async () => {
+    await deployContract(CONTRACT_PATH, [], { silent: true }).should.be
+      .fulfilled;
+    await deployContract(CONTRACT_PATH, [], { silent: true }).should.be
+      .fulfilled;
+  });
+
+  it('should reject dummy file', async () => {
+    await deployContract(DUMMMY_FILE, [], { silent: true }).should.be.rejected;
+  });
+
+  it('should reject flattend contract', async () => {
```

Comment:
I think test msg should be changed like:
"should reject when contract name and file name are different"